man/tpm2_createak.1.md

```diff
@@ -43,8 +43,9 @@ loaded-key:
     Specifies the current owner authorization.
     Same formatting as the endorse password value or **-e** option.
 
-  * **-E**, **--ek-handle**=_EK\_HANDLE_:
-    Specifies the persistent handle of the EK.
+  * **-C**, **--ek-context**=_EK\_CONTEXT\_OBJECT_:
+    Specifies the object context of the EK. Either a file or a handle's hex id.
```

Comment:
We support arbitrary bases for the handle number as understood by strtoul with a base of 0.